Creating-Objects.md

```diff
@@ -9,16 +9,16 @@
 
 ## Introduction
 
-When collecting user input, we often define the kind of input we expect using [[schemas|Schemas]]. These schemas can describe arbitrarily nested collections of base types. We call these base types _objects_. For example, consider this schema:
+When collecting user input, we often define the kind of input we expect using [[schemas|Schemas]]. These schemas can describe arbitrarily nested collections of simple types. We call these simple types _objects_. For example, consider this schema:
```

Comment:
I think this -- "These schemas can describe arbitrarily nested collections of simple types. We call these simple types _objects_" -- is still incorrect. 

An object is basically a custom type. It doesn't really matter whether it's arbitrary nested or simple or whatever ... some objects are simple and some aren't, some objects are nested and some aren't. It's just a custom type, typically with its own frontend viewer/editor component (see extensions/objects).

Reply:
fixed